pkg/network/ovs.go

```diff
@@ -15,8 +16,12 @@ import (
 
 func changeSafeOvs(prev, next *opv1alpha1.NetworkAddonsConfigSpec) []error {
```

Comment:
We can drop this function completely. All changes to the ovs value are not valid. There are only two options, it is there or it is not.

Reply:
if  remove it , need to remove ovs_test.go totally, because this is all it test there
on it, unless you say otherwise